ds3-sdk/src/main/java/com/spectralogic/ds3client/commands/interfaces/RequestHeadersImpl.java

```diff
@@ -95,7 +92,11 @@ public boolean containsKey(final String key) {
      */
     @Override
     public Multimap<String, String> getMultimap() {
-        return headers;
+        final ImmutableMultimap.Builder<String, String> builder = ImmutableMultimap.builder();
```

Comment:
The builder has a method: https://google.github.io/guava/releases/snapshot/api/docs/com/google/common/collect/ImmutableMultimap.Builder.html#putAll-com.google.common.collect.Multimap-

that will copy the whole contents of a multimap, you shouldn't have to manually perform the copy.